site/components/Input/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { InputHTMLAttributes } from "react";
+import styled from "@emotion/styled";
+import * as typography from "@klimadao/lib/theme/typography";
+import breakpoints from "@klimadao/lib/theme/breakpoints";
+
+const StyledInput = styled.input<InputProps>`
+  ${typography.caption}; // TODO: not working, syntax not compatible with styled?
```

Comment:
Any ideas on why this css interpolation doesn't work? :/

Reply:
Also. I could be wrong but `styled` miight need special babel transpilation magic, unlike `css` which we can safely use anywhere including `@klimadao/lib`

Reply:
No strong opinions here. I've collaborated on design systems using the following pattern above.
I'd love to learn and work out how to similar thing using css composition instead!